hack/openshift/fetch-tektoncd-catalog-tasks.sh

```diff
@@ -142,7 +142,7 @@ main() {
 
   case "$type" in
     "default")
-      dest_dir=${1:-'cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_external'}
+      dest_dir=${1:-'cmd/openshift/operator/kodata/tekton-addon/addons/02-tasks/source_external'}
```

Comment:
I think we no more require default case now because we don't have 02-tasks
cc @jkandasa 

May be @pratap0007 you can work on this script modification as a follow up PR